### ISSUE TITLE
Python client

### ISSUE THREAD

**Issue by killme2008** [GreptimeTeam/greptimedb#1180]
### What type of enhancement is this?

User experience

### What does the enhancement do?

We need a python client for GreptimeDB.

We already have:
* [Java client](https://github.com/GreptimeTeam/greptimedb-client-java)
* [Golang client](https://github.com/GreptimeTeam/greptimedb-client-go)

And there is a doc describing how to write a gRPC SDK for GreptimeDB:

https://docs.greptime.com/developer-guide/how-to/how-to-write-sdk

**Comment by waynexia** [comment]
We can consider build one on top of Rust client (which you forget to mention) using pyo3

**Comment by killme2008** [comment]
@waynexia Yep, but we first have to extract the rust client as an independent project.

**Comment by waynexia** [comment]
cross referencing #606
